warehouse/templates/manage/projects.html

```diff
@@ -27,7 +27,11 @@ <h1 class="page-title">Your projects ({{ projects|length }})</h1>
     <div class="package-snippet">
       <div class="split-layout split-layout--table split-layout--wrap-on-tablet">
         <div>
-          <h3 class="package-snippet__title">{{ project.name }}</h3>
+          <h3 class="package-snippet__title">{{ project.name }}
+            {% if project.name in projects_sole_owned %}
+            <span class="badge package-snippet__sole-owner-badge">Sole owner</span>
```

Comment:
Can you wrap this in a link to the collaboration management page?

```html
<a href="{{ request.route_path('manage.project.roles', project_name=project.normalized_name) }}">
  <span class="badge package-snippet__sole-owner-badge">Sole owner</span>
</a>
```

Reply:
Updated